ew/static/slimeoid.py

```diff
@@ -20,6 +20,7 @@
             "a"
         ],
         str_create="You press a button on the body console labelled 'A'. Through the observation port, you see the rapidly congealing proto-Slimeoid take on a roughly teardrop-shaped form.",
+        str_create_nega="You move the planchette to spell \"BODY\", then move the planchette to \"A\". In front of you, you see the undulating proto-Negaslimeoid take on a roughly teardrop-shaped form.",
```

Comment:
i totally understand that coming up with a full new set of negaslimeoid parts for each of these is a lot to ask. but if people make negaslimeoids and we change the descriptions later, we'll essentially be retconning their slimeoids. not mandatory, but lmk if you want help/someone to write those, i know i'd be down to create nega variations, though it'd take awhile lol.